Core/Services/AuthenticationService.cs

```diff
@@ -20,17 +20,20 @@ public class AuthenticationService : IAuthenticationService
         private readonly IMapper _mapper;
         private readonly IJwtService _jwtService;
         private readonly IRepository<RefreshToken> _refreshTokenRepository;
+        private readonly IRepository<User> _userRepository;
```

Comment:
Why is this field here? It is not used anywhere in this file, except the 30 and 36 lines where its initialization is.

Reply:
Something strange, cause I commited to remove this field yesterday, so I don`t have this on my side and Visual Studio doesn`t see any changes